reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -501,8 +502,21 @@ class StaticHtmlReporter : Reporter() {
                     dl {
                         dd {
                             row.detectedLicenses.forEach { (finding, excludes) ->
+                                val firstFinding = finding.locations.first()
+                                val permalink = VcsHost.toPermalink(row.vcsInfo.copy(path = firstFinding.path),
+                                    firstFinding.startLine, firstFinding.endLine)
+
                                 if (excludes.isEmpty()) {
-                                    div { +finding.license }
+                                    div {
+                                        if (permalink != null) {
+                                            a {
+                                                href = permalink
+                                                +finding.license
+                                            }
+                                        } else {
+                                            +finding.license
```

Comment:
I hope this won't lead to the impression that the linked finding is the only finding, what do you think?
I also don't have a proposal I'm really convinced of. One idea is to show the first e.g. 3 locations and then add a '...' below.

Reply:
> I hope this won't lead to the impression that the linked finding is the only finding, what do you think?

Yeah, that's the potential risk I was aware of and tried to deal with in the commit message. I still believe listing all findings inline takes up too much space in many cases, but how about not linking the license name itself, but append some text like " (permalink to the first of X findings for this license)" and link the word "permalink"?

Reply:
I like your proposal.